slurm/05_rerun_by_timepoint.sh

```
@@ -0,0 +1,94 @@
+# Rerun remaining timeout jobs split by individual timepoints
+# Each job now runs just 1-2 timepoints instead of all 8
+#
+# Timeout jobs that need rerunning:
+# constRt high: GT=2 INC=5, GT=2 INC=6, GT=3 INC=3, GT=4 INC=4 (4 combos × 8 tp = 32)
+# incdecRt dec: GT=2 INC=4, GT=2 INC=5 (2 combos × 8 tp = 16)
+# casestudy: GT=2 INC=6 tp5-8 (1 combo × 4 tp = 4) - split into individual
+# Plus casestudy GT=4 INC=6 tp5-8 (1 combo × 4 tp = 4)
+# Total: 32 + 16 + 4 + 4 = 56 jobs
+
+if [[ "$SLURM_SUBMIT_DIR" == */slurm ]]; then
+    cd "$SLURM_SUBMIT_DIR/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)